crypto/aes/aes_local.h

```diff
@@ -24,6 +24,7 @@
 #  define PUTU32(ct, st) { (ct)[0] = (u8)((st) >> 24); (ct)[1] = (u8)((st) >> 16); (ct)[2] = (u8)((st) >>  8); (ct)[3] = (u8)(st); }
 # endif
 
+typedef unsigned long long u64;
```

Comment:
Better to use the `uint64_t` type which OpenSSL ensures is defined properly in `e_os2.h`.  This could conflict with the many _uLL_ constants.

Reply:
In the next line there is a u32.
Do you see a problem with the LL constants?

Reply:
I did wonder about them.  However, we've got them elsewhere in the code (e.g. curve448) so I'm guessing that we accept them now.  This would also imply that _long long_ is okay.

Reply:
I agree with Pauli that ```uint64_t``` is preferred over ```unsigned long long``` directly.  Constants are a bit awkward (and at least in C90 the compiler is supposed to pick the type automatically to fit the value), so portable code with strict compiler warnings may not be able to avoid casts.

Reply:
It seems that agreement has been reached about this using the `uint64_t` type instead of `unsigned long long` (which BTW omits the final `int`).

In the past 64 bit integer constants have been wrapped up in macros that are given two 32 bit arguments.